LiveSplit.AutoSplitters.xml

```diff
@@ -19505,4 +19505,32 @@
         <Type>Script</Type>
         <Description>Door splitter and Item Splitter are available. (By TheDementedSalad)</Description>
     </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>25 To Life</Game>
+            <Game>TTL</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/LeonSReckon/TTL-ASL/main/25%20To%20Life.asl</URL>
+        </URLs>
+        <Type>Script</Type>
+        <Description>Auto Start/Split/Reset and Load Remover are Available (By LeonSReckon)</Description>
+    </AutoSplitter>
+    <AutoSplitter>
+        <Games>
+            <Game>Gears of War</Game>
+            <Game>Gears of War for Windows</Game>
+            <Game>Gears of War GFWL</Game>
+            <Game>Gears of War Game For Windows Live</Game>
+            <Game>Gears of War 2006</Game>
+            <Game>Gears of War (2006)</Game>
+            <Game>Gears of War 1</Game>
+            <Game>Gears of War: 1</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/LeonSReckon/Gears-of-War-1-ASL/main/Gears%20Of%20War%201.asl?token=GHSAT0AAAAAACHPAJJXB65VUE2NKIWVKFIMZIACYFA</URL>
```

Comment:
This link leads to a 404 error.

Reply:
I filled the check list
and I changed the link. For some reason this raw specifically creates a weird link

Reply:
The link stopped working again I think the ASL/Repository file has a problem

Reply:
Is the repository not public?

Reply:
I fixed the problem
I created a new repository